.github/workflows/format.yml

```diff
@@ -16,3 +16,6 @@ jobs:
           python-version: "3.12"
       - name: Ruff Format and Lint Check
         uses: chartboost/ruff-action@v1
+        with:
+          version: 0.5.7
+          args: "format --check"
```

Comment:
@jamshale PS: I think this action was just doing `ruff check`, since it didn't include `args: "format --check"`. 
https://github.com/ChartBoost/ruff-action?tab=readme-ov-file#use-ruff-format

Reply:
This checks linting and formatting now? I thought I had tested this but might have just been checking linting.